TrafficCapture/build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'jacoco'
     id 'org.owasp.dependencycheck' version '8.2.1'
+    id 'io.freefair.lombok' version '8.6' apply false
```

Comment:
Why is the lombok plugin being defined at this level?  I suppose that it's to establish the version for all of the subprojects to use.  Why/how does that work (trying to learn gradle some more)

Reply:
Gradle treat plugins differently from how it treats dependencies.  In this case by supplying the version, but not applying it the subprojects all pull in the same version.

Note; there is a similar issue with annotationProcessor which even though they look like a dependency that aren't the same.  Leaky abstraction :/